controllers/auth_config_controller.go

```diff
@@ -179,24 +181,30 @@ func (r *AuthConfigReconciler) translateAuthConfig(ctx context.Context, authConf
 	}
 
 	for identityCfgName, identity := range authConfigIdentityConfigs {
-		extendedProperties := make([]evaluators.IdentityExtension, len(identity.Defaults)+len(identity.Overrides))
+		extendedProperties := make([]evaluators.IdentityExtension, 0)
 		for propertyName, property := range identity.Defaults {
-			extendedProperties = append(extendedProperties, evaluators.NewIdentityExtension(propertyName, json.JSONValue{
-				Static:  property.Value,
-				Pattern: property.Selector,
-			}, false))
+			if value, err := stringValueFrom(&property); err != nil {
```

Comment:
Same as https://github.com/Kuadrant/authorino/pull/495/files#r1821989655. The value of an extended property can be of any type.

Reply:
Also, just to be clear (and we may want to rename that function), this does return a "JSON literal", but for `String` returns a plain string... i.e. w/o the quotes.